src/main/webapp/ChapMap.js

```diff
@@ -69,13 +75,27 @@ class ChapMap {
    * Adds click listeners to the map customization buttons.
    * A client can only add markers when the "adding markers" mode is on
    */
-  addBtnListeners_(viewBtn, addMarkerBtn, chatBtn, backBtn) {
+  addBtnListeners_() {
+    let addMarkerBtn = this.getEl_("addMarkerBtnWrapper");
+    let backBtn = this.getEl_("backBtnWrapper");
+    let viewBtn = this.getEl_("viewBtnWrapper");
+    let chatBtn = this.getEl_("chatBtnWrapper");
+
     addMarkerBtn.addEventListener('click', () => this.enableAddingMarkers_());
     viewBtn.addEventListener('click', () => this.disableAddingMarkers_());
     chatBtn.addEventListener('click', () => toggleChat());
     backBtn.addEventListener('click', () => window.location='/');
   }
 
+  /**
+   * @Private
+   * Returns the element with the given id.
+   * @param {String} id the id of the DOM element
+   */
+  getEl_(id) {
```

Comment:
nit: document.getElementById is a common function that everyone is familiar with. While it is a little longer, the code can be understood faster without adding this function.

Reply:
That makes sense, I'll revert the change. I was planning on making a helper function for creating DOM elements where you can pass the element type, it's id, and classname as parameters since this would shorten up the code wherever I need to add HTML dynamically (like in the map-sharing popup, chat, welcome page, and information windows). Should I go ahead with it or would it be counter-productive in terms of making the code more readable?

Reply:
Sure you can add a helper method for that. My comment was mainly because the method is only one line, so it felt unnecessary.

Reply:
Ahh I understand, I'll keep that in mind!